app/scripts/forms/PlaceForm.js

```diff
@@ -31,6 +31,14 @@ const validate = values => {
     )
   }
 
+  if (!values.accessibilityInfo) {
+    errors.accessibilityInfo = translate('forms.place.errors.accessibilityInfoRequired')
```

Comment:
You're checking against a minimum length on the server side, I would add that here as well to make sure you're not causing an server error.

Reply:
https://github.com/adzialocha/hoffnung3000/pull/94/commits/5dd7a30f35553add1432d56c69618f2fd5617e4d